streamwebs_frontend/streamwebs/templates/streamwebs/sites.html

```diff
@@ -0,0 +1,169 @@
+{% extends "streamwebs/base.html" %}
+
+{% load staticfiles %}
+
+{% block title %}Sites - Streamwebs{% endblock %}
+
+{% block scripts %}
+<script src="http://maps.google.com/maps/api/js"></script>
+<script type="application/javascript">
```

Comment:
You should factor it out to it's own file. Really all of our custom JS should be in it's own file so the user can cache it on their end and we don't need to worry about looking through all of our templates to edit dynamic functionality.